identity-apps-core/apps/authentication-portal/src/main/webapp/domain.jsp

```diff
@@ -28,14 +28,17 @@
 
 <%
     String domainUnknown = AuthenticationEndpointUtil.i18n(resourceBundle, "domain.unknown");
-    String errorMessage = AuthenticationEndpointUtil.i18n(resourceBundle, "authentication.failed");
+    String errorMessage = AuthenticationEndpointUtil.i18n(resourceBundle, "authentication.failed.please.retry");
     boolean loginFailed = false;
     if (Boolean.parseBoolean(request.getParameter("authFailure"))) {
         loginFailed = true;
         if (request.getParameter("authFailureMsg") != null) {
-            errorMessage = request.getParameter("authFailureMsg");
-
-            if (domainUnknown.equalsIgnoreCase(errorMessage)) {
+            String error = request.getParameter("authFailureMsg");
```

Comment:
Let's encode the params

Reply:
this is encoded here: https://github.com/wso2/identity-apps/pull/6488/files#diff-d105e0cbec2bb63d0ec08d147e8666c534ba8695e6d8a8071f0e2c9f615ded0bR87

Reply:
It is encoded for html content in that place. We need to encode for java here, before using the param value.

Reply:
updated with [f4fc9cd](https://github.com/wso2/identity-apps/pull/6488/commits/f4fc9cd042c23e91a09278eea5ed93d8e4ef8eb9)